_data/en/site.yml

```diff
@@ -0,0 +1,32 @@
+title: Specifying JavaScript.
+navigation:
+  TC39:
+    title: TC39
+    agenda: Upcoming Agenda
+    notes: Recent Meeting Notes
+    sub-menu: Toggle sub-menu for TC39
+  specs:
+    title: Specs
+    proposals: Active Proposals
+    ecma-262: ECMA-262, ECMAScript
+    ecma-402: ECMA-402, Intl API
+    sub-menu: Toggle sub-menu for specs
+  contribute:
+    title: Contribute
+    proposals: Contribute to Proposals
+    tests: Contribute to Tests
+    read: Reading the Spec
+    discourse: Join the Discourse
+    sub-menu: Toggle sub-menu for Contribute
+  state-of-proposals: State Of Proposals
```

Comment:
```suggestion
  state-of-proposals: State of Proposals
```

Reply:
Short prepositions are usually not capitalized, so I'll skip this one.

Reply:
We are in agreement with my suggestion. :)

See: https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

Reply:
ah, i read it backwards